Amit/index.js

```diff
@@ -1,37 +1,37 @@
 // q 1
 let sells = 6450;
 let cost = 15;
-let sold = sells / cost;
-console.log("Tickes sold", sold); // 430
+let tickets = sells / cost;
+console.log("Tickes sold", tickets); // 430
 
 // console.log(`${sold} bought tickets`);
 
 // 2
-let doller = 500;
-let week = 52;
-let mul = doller * week;
-console.log(mul);
+let income = 500;
+let weeks = 52;
+let totalIncome = income * weeks;
+console.log(totalIncome);
 
 // 3
 let a = 17;
 let b = 30;
-let result = (17 / 30) * 100;
+let percentage = (17 / 30) * 100;
 
-console.log(result);
+console.log(percentage + "%");
 
 //4
-let weidth = 4.75;
-let square = weidth * 4; // weidth+weadth+weidth+weidth
-console.log(square);
+let width = 4.75;
+let perimeterSquare = width * 4; // weidth+weadth+weidth+weidth
+console.log(perimeterSquare);
 
 //5
-let one = 5;
-let two = 6;
-let three = 7;
-perimeter = one * two * three;
+let sideOne = 5;
+let sideTwo = 6;
+let sideThree = 7;
+let perimeter = sideOne * sideTwo * sideThree;
```

Comment:
In 5th question you have to addition and not multiplication! to calculate the perimeter of a triangle
let perimeter = sideOne + sideTwo + sideThree;

Reply:
done